### ISSUE TITLE
Prediction For the Sample Images

### ISSUE THREAD

**Issue by Guardian99** [sergiomsilva/alpr-unconstrained#125]
After running the command it gets executed with the final line being Performing OCR... and it exits then. In the output folder specified there are a few images and a CSV File but that CSV files does not contain the detected License Plate Number. 
Could someone please tell where is the prediction for the sample images stored or if i am some missing some step not mentioned in the repository. 
Any help would be really appreciated.

**Comment by khawar512** [comment]
Do you use Ubuntu OS?

**Comment by Guardian99** [comment]
Yes,I do. The environment is similar to the one mentioned in the repository. The code gets executed but I am unable to get the prediction out of the test sample images. While training , it shows 0 cars found and at the end of it OCR done.

**Comment by khawar512** [comment]
Could you please see images?. You have to go tmp directory where you will found all detected images.

**Comment by Guardian99** [comment]
There are no detected images. The model does not predict anything. It simply mimics the input image. If you could read my doubt again and tell if you were able to run this project from detection to prediction and results then it would be really helpful.

**Comment by muhammadabdullah34907** [comment]
@Guardian99 
I was also dealing with same problem and then I test it on python2.7 by creating virtual environment. All works well. Just change the python version and it will work fine.

**Comment by sinclairnick** [comment]
If your problem is with "0 cars detected", change this line
```python
R = [r for r in R if r[0] in ['car', 'bus']]
```
to
```python
R = [r for r in R if r[0] in [b'car', b'bus']]
```

**Comment by C0NGTRI123** [comment]
> 

it working for me! Thank you

**Comment by C0NGTRI123** [comment]
@sinclairnick 
It working for me. Thank you!
